ci.nix

```diff
@@ -13,6 +13,8 @@ pkgs.nix-community-infra // rec {
   build01-system = build01.system;
   build02 = importNixOS ./build02/configuration.nix "x86_64-linux";
   build02-system = build02.system;
-  build03 = importNixOS ./build02/configuration.nix "x86_64-linux";
+  build03 = importNixOS ./build03/configuration.nix "x86_64-linux";
   build03-system = build03.system;
+  build04 = importNixOS ./build04/configuration.nix "aarch64-linux";
```

Comment:
Now that there's a second node, you need to configure the binary cache in binary-caches.json, so that drvs can be shared between nodes.

Reply:
Yeah. We noticed this the hard way.

Reply:
I'd like to make it more flexible, but right now the focus is on more high level aspects like a new nix file (including flakes support) and more "typical" CI/CD features.

Reply:
It's got green https://hercules-ci.com/github/nix-community/infra/jobs/13 eventually.